### PR TITLE
Added an overlay example to docs

### PR DESCRIPTION
**Pull request by Teuns**
The changes made to this pull request are quite simple; an example to trigger an overlay has been added. The overlay contains a panel. The example also includes the button to open the overlay.

### CODE REVIEW

**Comment by basmilius** [comment]
Thanks for the changes @Teuns. Please update the title and description of this pull request and provide information about what you did.

File: packages/docs/package.json [hunk 1]
```diff
@@ -38,4 +38,4 @@
 		"last 2 versions",
 		"not ie <= 10"
 	]
-}
+}
```
Comment (basmilius) [inline packages/docs/package.json]: Empty line is missing here. All files should end with an empty line

File: packages/docs/public/snippets/components/overlay/basic.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<button class="btn btn-contained btn-primary" data-action="latte:ui:overlay" data-name="latte-overlay">
+  	<span>Open</span>
+</button>
+
+<latte-overlay name="latte-overlay">
+  	<div class="panel">
+		<div class="panel-header">
+		  <h2 class="panel-title">Overlay Title</h2>
```
Comment (Droppers) [inline packages/docs/public/snippets/components/overlay/basic.html]: Mixed spaces and tabs

**Comment by basmilius** [comment]
Ok, everything seems fine to me!

**Pull request merged**